plugins/aws/sqs/queueUnprocessedMessages.js

```diff
@@ -0,0 +1,86 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'SQS Queue Unprocessed Messages',
+    category: 'SQS',
+    description: 'Ensures that Amazon SQS queue has not reached unprocessed messages limit.',
+    more_info: 'Amazon SQS queues should have unprocessed messages less than the limit to be highly available and responsive.',
+    recommended_action: 'Set up appropriate message polling time and set up dead letter queue for Amazon SQS queue to handle messages in time',
+    link: 'https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/working-with-messages.html',
+    apis: ['SQS:listQueues', 'SQS:getQueueAttributes', 'STS:getCallerIdentity'],
+    settings: {
+        unprocessed_messages_limit: {
+            name: 'SQS Queue Unprocessed Messages Limit',
+            description: 'Maximum allowed limit for SQS queue unprocessed messages',
+            regex: '^[0-9]{1,3}',
+            default: '100'
```

Comment:
where did you get the default of 100? it seems low to me for an enterprise account